src/components/IndicatorBar.vue

```diff
@@ -1,24 +1,38 @@
 <script setup lang="ts">
 import { computed } from "vue"
+import { useStore } from "../store"
 
 const props = defineProps<{
   title: string
   value: number
 }>()
 
+const emit = defineEmits<{
+  (event: "change", value: number): void
+}>()
+
+const store = useStore()
+const devMode = computed(() => store.state.devMode)
+
 const indicatorStyle = computed(() => {
   const backgroundColor = props.value > 40 ? "green" : props.value > 20 ? "goldenrod" : "red"
   return {
     width: props.value + "%",
     backgroundColor,
   }
 })
+
+function handleClick(event: MouseEvent) {
+  if (devMode.value) {
+    emit("change", event.offsetX) // requires the clientWidth of the element to be exactly 100
+  }
+}
```

Comment:
Ah! I get it: Interactive test tool. Sweeet idea!

Unfortunately the connection to the game values seems broken: At the start "Beliebtheit" shows full green, "Finanzen" und "CO2-Budget" show completely white bar (and the corresponding images). See my screen shots above. That did not change.

Reply:
That was indeed not intended. I'll see how this can be solved.